src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double sideLength;
+
+    public Square(String color, double sideLength) {
+        super(color);
+        this.sideLength = sideLength;
+    }
+
+    @Override
+    public double area() {
+        return sideLength * sideLength;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Square color: " + getColor()
+                + ", Side length = " + sideLength
+                + "Area = " + area());
```

Comment:
There is a missing comma in the `draw` method's output between the side length and the area. This can lead to unclear output. Consider adding a comma for better readability.